config/openapi/vendor-api-v1.1.yml

```diff
@@ -0,0 +1,1780 @@
+        hesa_degest:
+          type: string
+          description: HESA code for degree establishment
+          example: '0002'
+          maxLength: 20
+          nullable: true
+        hesa_degctry:
+          type: string
+          description: HESA code for degree country
+          example: XF
+          maxLength: 2
+          nullable: true
+        hesa_degstdt:
+          type: string
+          description: Degree start date in ISO8601 format
+          format: date
+          example: 2019-09-18
+          maxLength: 10
+          nullable: true
+        hesa_degenddt:
+          type: string
+          description: Degree end date in ISO8601 format
+          format: date
+          example: 2019-09-18
+          maxLength: 10
+          nullable: true
+    Reference:
+      type: object
+      additionalProperties: false
+      required:
+      - id
+      - name
+      - email
+      - relationship
+      - reference
+      - referee_type
+      - safeguarding_concerns
+      properties:
+        id:
+          type: integer
+          description: A unique reference ID
+          example: 4321
+        name:
+          type: string
+          description: The referee’s name
+          maxLength: 200
+          example: Julia Wild
+        email:
+          type: string
+          description: The referee’s email
+          maxLength: 100
+          example: julia@example.com
+        relationship:
+          type: string
+          description: |
+            The candidate’s description of their relationship with the referee
+          maxLength: 10240
+          example: Julia was my mentor at Cheadle Hulme High
+        reference:
+          type: string
+          description: The reference content provided by the referee.
+          maxLength: 10240
+          example: Boris is committed to the profession of teaching...
+        safeguarding_concerns:
+          type: boolean
+          description: |
+            Whether the referee flagged safeguarding concerns about this candidate
+          example: true
+        referee_type:
+          description: |
+            The referee’s relationship to the candidate
+          type: string
+          example: professional
+          enum:
+          - academic
+          - professional
+          - school_based
+          - character
+    Rejection:
+      type: object
+      additionalProperties: false
+      required:
+      - reason
+      - date
+      properties:
+        reason:
+          type: string
+          description: The reason for rejection or offer withdrawal. If none are yet provided for an application rejected by default, the value `Not entered` is returned
+          maxLength: 65535
+          example: Does not meet minimum GCSE requirements
+        date:
+          type: string
+          format: date-time
+          description: Time of the rejection or offer withdrawal
+          example: 2019-09-18T15:33:49.216Z
+    ReinstateOffer:
```

Comment:
Unconditional offers (no conditions specified) cause an `offer` status to change straight into a `recruited` state when the candidate accepts an offer (rather than `pending_conditions`). At the point of deferral, applications are either in `pending_conditions` or `recruited` status, both of which are past `offer`. When re-instating all providers need to indicate is the desired new status, and this is already explained.